keyboards/albert_smart/albert_smart84/config.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2024 albert-zheng
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+
+#define DRIVER_COUNT 1
+
+#define RGB_MATRIX_KEYPRESSES // reacts to keypresses
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS // enable framebuffer effects
+#define RGB_MATRIX_SLEEP // USB待机时候停止灯效
+#define RGB_MATRIX_SOLID_REACTIVE_GRADIENT_MODE
+#define RGB_MATRIX_TIMEOUT 0 //#define RGBLIGHT_DISABLE_KEYCODES
```

Comment:
This all should be moved to the info.json.  `qmk info -f json` can help with that.

Reply:
okay